src/jit/gentree.cpp

```diff
@@ -10694,7 +10694,8 @@ void Compiler::gtDispConst(GenTree* tree)
         case GT_CNS_INT:
             if (tree->IsIconHandle(GTF_ICON_STR_HDL))
             {
-                printf(" 0x%X \"%S\"", dspPtr(tree->gtIntCon.gtIconVal), eeGetCPString(tree->gtIntCon.gtIconVal));
+                const wchar_t* value = eeGetCPString(tree->gtIntCon.gtIconVal);
+                printf(" 0x%X \"%S\"", dspPtr(tree->gtIntCon.gtIconVal), value ? value : W("(null)"));
```

Comment:
The problem is not here, but in the printf. Printf itself should handle the NULL string pointer this way. I guess it was introduced by my recent PAL cleanup where I have rewired some of the printf functionality to a different implementation in the PAL. Let me fix it.